CMSgov/easi-app
# [NOREF] Remove null types from cedar core

# NOREF

## Changes and Description

The code generator for the client defaults to zero values, so there's not much use wrapping them in the `null` package types.

- Removes `null` package types from CEDAR core client. 
- Replaces nil pointer dereferences with `zero.String` types for safety in the client.

## How to test this change

- Run the test suites
- Use Postman to test CEDAR resolvers
- Use client to check Systems tab content

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
